ansible_collections/arista/avd/roles/eos_designs/templates/network_services/vlan-interfaces.j2

```diff
@@ -75,6 +75,25 @@ vlan_interfaces:
         vrf: {{ svi_ip_helpers[helper_ip].source_vrf | arista.avd.default }}
 {%                 endfor %}
 {%             endif %}
+{%             if svi.ospf.enabled is arista.avd.defined(true) and vrf.ospf.enabled is arista.avd.defined(true) %}
+    ospf_area: {{ svi.ospf.area | arista.avd.default(0) }}
+    ospf_network_point_to_point: {{ svi.ospf.point_to_point | arista.avd.default(false) }}
+{%             endif %}
+{%             if svi.ospf.cost is arista.avd.defined %}
+    ospf_cost: {{ svi.ospf.cost }}
+{%             endif %}
+{%             if svi.ospf.authentication is arista.avd.defined("simple") and svi.ospf.simple_auth_key is arista.avd.defined %}
+    ospf_authentication: "simple"
+    ospf_authentication_key: {{ svi.ospf.simple_auth_key }}
+{%             elif svi.ospf.authentication is arista.avd.defined("message-digest") and svi.ospf.message_digest_keys is arista.avd.defined %}
+    ospf_authentication: "message-digest"
+    ospf_message_digest_keys:
+{%                 for key in svi.ospf.message_digest_keys | arista.avd.natural_sort %}
+      {{ key.id }}:
```

Comment:
no variable protection for `key.id` and `key.key`

Reply:
Fixed.